### PR TITLE
TECH-661: Move test graphqlRender from JUnit to Cypress

### PR DESCRIPTION
**Pull request by jayblanc**
## JIRA

https://jira.jahia.org/browse/TECH-661

## Description

Create a site to test if some nodes have the property isDisplayableNode or not. That property should be only some nodes like pages and not on others like list. 

## Tests

The following are included in this PR
- [X] Unit Tests (Most changes _should_ have unit tests)
- [ ] Integration Tests

## Checklist

I have considered the following implications of my change: 

- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
- [ ] Performance
- [ ] Migration
- [ ] Code maintainability

## Documentation

- [ ] Inline documentation
- [ ] Internal Documentation (wiki)
- [ ] User-facing Documentation

### CODE REVIEW

File: tests/provisioning-manifest-build.yml [hunk 1]
```diff
@@ -1 +1,2 @@
 - sleep: 10
+- include: 'mvn:org.jahia.packages/jahia-additional-modules/${const:org.jahia.api.Constants.JAHIA_PROJECT_VERSION}/yaml/provisioning'
```
Comment (Fgerthoffert) [inline tests/provisioning-manifest-build.yml:2]: Why running the provisioning for "jahia-additional-modules" ?

File: tests/provisioning-manifest-snapshot.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- include: 'mvn:org.jahia.packages/jahia-additional-modules/${const:org.jahia.api.Constants.JAHIA_PROJECT_VERSION}/yaml/provisioning'
```
Comment (Fgerthoffert) [inline tests/provisioning-manifest-snapshot.yml:1]: Same question than above

**Review by Fgerthoffert — COMMENTED** [review]
You also probably need to remove the test you did migrate over

File: tests/provisioning-manifest-build.yml (hunk 1 above)
Comment (jayblanc) [reply tests/provisioning-manifest-build.yml:2]: Test is creating a site using a template that is in a module

**Review by Fgerthoffert — APPROVED** [review]

**Pull request merged**